applications/app/views/fragments/galleryHeader.scala.html

```diff
@@ -9,7 +8,7 @@
     <h1 class="is-hidden" itemprop="headline">@Html(gallery.item.trail.headline)</h1>
 
     <div class="gs-container">
-        @if(gallery.showBadge || staticBadgesSwitch.isSwitchedOn) {
+        @if(gallery.showBadge) {
```

Comment:
I think this condition can be removed as it was having no effect when switch was on.